ksqldb-serde/src/test/java/io/confluent/ksql/serde/FormatFactoryTest.java

```diff
@@ -135,10 +135,15 @@ public void shouldThrowWhenCreatingFromUnsupportedProperty() {
   @Test
   public void shouldNotThrowWhenCreatingFromSupportedProperty() {
     // Given:
-    final FormatInfo formatInfo = FormatInfo.of("AVRO", ImmutableMap.of("fullSchemaName", "1"));
+    final FormatInfo formatInfo = FormatInfo.of("AVRO", ImmutableMap.of("schemaId", "1", "fullSchemaName", "2"));
+    final FormatInfo protobufInfo = FormatInfo.of("PROTOBUF",
+        ImmutableMap.of("schemaId", "1"));
+    final FormatInfo jsonSRFormatInfo = FormatInfo.of("JSON_SR",
+        ImmutableMap.of("schemaId", "123"));
```

Comment:
Is "fullSchemaName" missing for protobuf and json?

Reply:
Let me add that